web/urls.py

```
@@ -4,6 +4,8 @@
 from django.contrib import admin
 from django.contrib.auth.decorators import login_required
 from django.urls import include, path
+from django.views.generic import RedirectView
```

Ruff (0.8.2)

`django.views.generic.RedirectView` imported but unused

Remove unused import: `django.views.generic.RedirectView`

(F401)